src/abstractdataframe/show.jl

```diff
@@ -203,38 +198,38 @@ julia> df = DataFrame(A = 1:3, B = ["x", "yy", "z"]);
 
 julia> maxwidths = DataFrames.getmaxwidths(df, stdout, 1:1, 3:3, :Row)
 3-element Array{Int64,1}:
- 1
- 1
+ 5
+ 6
  3
 
-julia> DataFrames.getchunkbounds(maxwidths, true, displaysize()[2])
-2-element Array{Int64,1}:
- 0
- 2
+julia> DataFrames.getchunkedcols(maxwidths, true, displaysize()[2])
+1-element Array{Array{Any,1},1}:
+ [1, 2]
 ```
 """
-function getchunkbounds(maxwidths::Vector{Int},
-                        splitcols::Bool,
-                        availablewidth::Int) # -> Vector{Int}
+function getchunkedcols(maxwidths::Vector{Int},
+                         splitcols::Bool,
+                         availablewidth::Int) # -> Vector{Int}
     ncols = length(maxwidths) - 1
     rowmaxwidth = maxwidths[ncols + 1]
     if splitcols
-        chunkbounds = [0]
+        chunkedcols = [[]] # begin with a single chunk with no columns
```

Comment:
This probably should be `[Int[]]` right as now you create `Array{Array{Any,1},1}`?